db/migrate/20230718135458_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.text :photo
+      t.text :bio
+
+      t.timestamps
+    end
+    add_index :users, :name
+  end
+end
```

Comment:
- [ ] You did a good job working on this milestone! 💯 However, according to the ERD:

![image](https://github.com/Elhambasir/blog-app/assets/77069011/ae9b3aaa-d9fa-4d0d-85a9-417fd844449f)

A User should have a  property `PostsCounter`. Please consider adding it! 🤓 🔧